Apps/WebClient/src/ClientApp/src/constants/dateTimeFormat.ts

```diff
@@ -0,0 +1,3 @@
+export default abstract class DateTimeFormatter {
```

Comment:
DateTimeFormat

Reply:
Weird, I thought I had checked it in, but it wasn't. Thanks @FuriousLlama for the nice catch!